payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionUI.kt

```diff
@@ -5,31 +5,15 @@ import androidx.annotation.StringRes
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Card
-import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.semantics.heading
 import androidx.compose.ui.semantics.semantics
-import androidx.compose.ui.text.font.FontWeight
-import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.dp
-import androidx.compose.ui.unit.sp
 import com.stripe.android.ui.core.PaymentsTheme
 
-/**
- * This is the style for the section title.
- *
- * Once credit card is converted use one of the default material theme styles.
- * TODO(skyler-stripe): remove this once we centralize fonts.
- */
-internal object SectionTitle {
```

Comment:
WHere did the title go?  It is used for the Card element to display "Card information" and "Billing details"